x/pylons/client/cli/query_list_item_by_owner_test.go

```diff
@@ -61,7 +61,7 @@ func TestListItemByOwner(t *testing.T) {
 				require.NoError(t, err)
 				var resp types.QueryListItemByOwnerResponse
 				require.NoError(t, net.Config.Codec.UnmarshalJSON(out.Bytes(), &resp))
-				//require.NotNil(t, resp.Items)
+				// require.NotNil(t, resp.Items)
```

Comment:
Is the commented code for testing purpose or just code that needs to be removed?

Reply:
I haven't commented it.  the software has formatted the commented code.